src/fastmcp/utilities/ui.py

```diff
@@ -454,7 +454,7 @@ def create_page(
     content: str,
     title: str = "FastMCP",
     additional_styles: str = "",
-    csp_policy: str = "default-src 'none'; style-src 'unsafe-inline'; img-src https:; base-uri 'none'",
+    csp_policy: str = "default-src 'none'; style-src 'unsafe-inline'; img-src https: data:; base-uri 'none'",
```

Comment:
I've also included it as a default here, given at least `create_unregistered_client_html()` would benefit from it (i.e. it also includes the server icon). I considered that there shouldn't really be any security ramifications of this so as a default it should be ok.